src/main/java/uk/gov/hmcts/reform/managecase/client/SystemUserAuthHeadersInterceptor.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.reform.managecase.client;
+
+import feign.RequestInterceptor;
+import feign.RequestTemplate;
+import uk.gov.hmcts.reform.managecase.security.SecurityUtils;
+
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+
+public class SystemUserAuthHeadersInterceptor implements RequestInterceptor {
+
+    public static final String SERVICE_AUTHORIZATION = "ServiceAuthorization";
```

Comment:
Perhaps move this to `SecurityUtils` so we don't have it defined in both `SystemUserAuthHeadersInterceptor` and `AuthHeadersInterceptor`?